docs/modules/ROOT/pages/integrating-with-existing-deployments.adoc

```diff
@@ -0,0 +1,100 @@
+
+Each section within these tutorials designed to be independent, so only follow the configuration steps required.
+
+== Pre-Requisites
+You will need to have Cluster Monitor installed.
+If this is not the case, please see our documentation on how to xref:tutorial-standalone.adoc#_install_cluster_monitor[install Cluster Monitor].
+
+== Prometheus
+
+Once you have completed all of the below steps you may need to restart Prometheus to apply the configuration changes.
+
+=== Configuring the Scrape config for your Couchbase cluster
+
+You will need to modify your existing Prometheus configuration file - this is typically named `prometheus.yml`.
+
+See our http://localhost:4040/cmos/0.1/tutorial-standalone.html#_install_prometheus[Install Prometheus] documentation for an example of what this file should look like with an appropriate Couchbase `scrape_config`.
+
+=== Install Prometheus Alerting Rules
+
+Follow the Standalone tutorial instructions on http://localhost:4040/cmos/0.1/tutorial-standalone.html#_prometheus_alerting_rules[Installing Prometheus Alerting rules].
+
+== Alertmanager
+
+=== Downloading and installing Alert templates
+
+Follow the instructions in the http://localhost:4040/cmos/0.1/tutorial-standalone.html#_prometheus_alerting_rules[Standalone example guide].
+
+== Grafana
+
+=== Install all required Grafana plugins
+
+The linked instructions for each plugin offer a Grafana cloud one-click install, a command-line based install for a running instance, or a `.zip` file which can be unpacked manually into your Grafana plugins directory. 
+If you prefer to use configuration files, or run in a Docker container, you may prefer an alternative install method (see below).
+
+. https://grafana.com/grafana/plugins/marcusolsson-json-datasource/[JSON API], which has associated https://grafana.com/grafana/plugins/marcusolsson-json-datasource/?tab=installation[installation instructions].
+
+==== Alternative install methods
+If you:
+
+- Provision Grafana v7.1+ through the use of configuration files, you can https://grafana.com/docs/grafana/latest/administration/provisioning/#plugins[specify the plugins to install in your configuration file]. 
+- Are running Grafana in a Docker container, you can simply pass through an additional environment variable `GF_INSTALL_PLUGINS=$PLUGIN_NAME $VERSION` where the `$PLUGIN_NAME` and latest `$VERSION` can be found on the plugin's homepage (linked under "installation instructions" for each). 
+
+=== Download and install CMOS dashboards
+
+The dashboards can be found in the official GitHub repository under https://github.com/couchbaselabs/observability/tree/main/microlith/grafana/provisioning/dashboards[microlith/grafana/provisioning/dashboards]/.
+
+Download them and copy them to a folder accessible by your Grafana installation. You will need to update the `path` argument in your Grafana configuration file - this is typically named `grafana.yml`.
```

Comment:
Can we be more specific - which object is `path` in (I'd be very surprised if there's only one thing in the config named `path`).

Also, the main Grafana config is `grafana.ini` - IIRC provisioning has a separate config, should we call that out?

Reply:
Have added some clarifying information.